### ISSUE TITLE
Write the content for networking service

### ISSUE THREAD

**Issue by 1Copenut** [1Copenut/c3-eleventy#4]
There 4-6 core services that should be written. This issue will capture the copy for accessibility.

- [ ] Write first draft
- [ ] Proofread (ideally with outside help)
- [ ] Add to Markdown file
